docs/package_info/glossary.rst

```diff
@@ -0,0 +1,46 @@
+Glossary
+========
+
+.. _access_token:
+
+* ``Access Token``: A token that is obtained from authenticating with Reddit.
+  The token lasts for an hour, so multiple tokens may need to be obtained.
+
+.. _camel_case:
+
+* ``Camel Case``: A type of variable notation where multiple words are
+  seperated by upper case. For example, the glossary entry for
+  :ref:`Access Token <access_token>` would be written as ``accessToken`` in
+  Camel Case.
```

Comment:
`accessToken` is "lower camel case." Camel case would be AccessToken.

That said, I'm not sure if this belongs in a PRAW glossary as it isn't related to Reddit or PRAW.

Reply:
I'll omit the data, but I was thinking for development. Maybe I should make a sub-glossary under the development files.